backend/handlers/handlers.go

```diff
@@ -339,22 +342,35 @@ func (handlers *Handlers) postAddAccountHandler(r *http.Request) (interface{}, e
 		return nil, err
 	}
 	keypath := signing.NewEmptyAbsoluteKeypath()
-	extendedPublicKey, err := hdkeychain.NewKeyFromString(jsonExtendedPublicKey)
-	if err != nil {
-		return map[string]interface{}{"success": false, "errorCode": "xpubInvalid"}, nil
-	}
-	if extendedPublicKey.IsPrivate() {
-		return map[string]interface{}{"success": false, "errorCode": "xprivEntered"}, nil
-	}
-	if btcCoin, ok := coin.(*btc.Coin); ok {
-		expectedNet := &chaincfg.Params{
-			HDPublicKeyID: btc.XPubVersionForScriptType(btcCoin, scriptType),
+
+	var configuration *signing.Configuration
+
+	if jsonAddress != "" {
+		if jsonCoinCode == "teth" || jsonCoinCode == "eth" {
+			if !common.IsHexAddress(jsonAddress) {
+				return nil, errp.WithStack(errors.ErrInvalidAddress)
```

Comment:
Smeonean invalid address? This leads to a crash in the frontend in webdev mode, and is silently ignored in the Qt app. See the examples below of how to send an error to the frontend, e.g.

```Go
 			return map[string]interface{}{"success": false, "errorCode": "invalidAddress"}, nil
```

(need to also add a i18n key in app/en.json for `invalidAddress`).